app/.eslintrc.js

```diff
@@ -108,6 +108,21 @@ module.exports = {
         module: "@arizeai/components",
         use: "@phoenix/components",
       },
+      {
+        name: "TextField",
+        module: "@arizeai/components",
+        use: "@phoenix/components",
+      },
+      {
+        name: "TextArea",
+        module: "@arizeai/components",
+        use: "@phoenix/components",
+      },
+      // {
+      //   name: "Form",
+      //   module: "@arizeai/components",
+      //   use: "@phoenix/components",
+      // },
```

Comment:
Did you mean to leave this?

```suggestion
```

Reply:
I'm really close on deprecating it 😓 - just a few more spots where there's layout implications